Cython/Includes/libcpp/bit.pxd

```diff
@@ -0,0 +1,31 @@
+cdef extern from "<bit>" namespace "std" nogil:
+    # bit_cast (gcc >= 11.0, clang >= 14.0)
+    cdef To bit_cast[To, From](From&)
+
+    # byteswap (C++23)
+    #cdef T byteswap[T](T)
+
+    # integral powers of 2 (gcc >= 10.0, clang >= 12.0)
+    cdef bint has_single_bit[T](T)
+    cdef T bit_ceil[T](T)
+    cdef T bit_floor[T](T)
+    cdef int bit_width[T](T)
+
+    # rotating (gcc >= 9.0, clang >= 9.0)
+    cdef T rotl[T](T, int)
+    cdef T rotr[T](T, int)
```

Comment:
It's often helpful to name arguments in order to support keyword argument calls. Here, users might want to pass the rotation bit count as keyword args (why not).